### PR TITLE
T37375-Create MetalLB docs

### PR DESCRIPTION
**Pull request by raquel-campuzano**

### CODE REVIEW

File: charts/metallb/_configure_layer2_mode.md.erb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+apps: all
+platforms: kubernetes
+id: configure_layer2_mode
+title: Configure MetalLB in layer 2 mode
+category: administration
+weight: 20
+highlight: 20
+=end %>
+
+The layer 2 mode is commonly used to make MetalLB behaves as a load-balancer. When configuring MetalLB in layer 2 mode, all traffic for a service IP goes to one node. From there, *kube-proxy* spreads the traffic to all the service's pods.
```
Comment (javsalgar) [inline charts/metallb/_configure_layer2_mode.md.erb:12]: behaves -> behave

File: charts/metallb/_configure_layer2_mode.md.erb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+This mode doesn't implement a load-balancer, it is most likely a failover mechanism: when one node is down, is automatically detected and the rest of the nodes will take over the ownership of the failed node IP addresses. 
+
+Find below an example of how to enable the layer 2 mode:
```
Comment (javsalgar) [inline charts/metallb/_configure_layer2_mode.md.erb:16]: I would clarify that the addresses in the `addresses` section is a subset of the available Service IPs the Kubernetes cluster has.

**Review by javsalgar — COMMENTED** [review]
LGTM, minor change

File: charts/metallb/_configure_layer2_mode.md.erb (hunk 2 above)
Comment (raquel-campuzano) [reply charts/metallb/_configure_layer2_mode.md.erb:16]: +1 Thanks for the clarification!

**Review by javsalgar — APPROVED** [review]

**Pull request merged**